cinema/views.py

```diff
@@ -167,6 +213,39 @@
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "movie",
+                type=str,
+                description="Filter by movie id",
+                required=False,
+            ),
+            OpenApiParameter(
+                "movie",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by movie id",
+            ),
+        ]
+    )
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type=str,
+                description="Filter by date",
+                required=False,
+            ),
+            OpenApiParameter(
+                "date",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by date",
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
```

Comment:
This is a code style issue. The `list` method is not doing anything beyond calling the superclass method. If you're not adding any additional logic, you can just omit this method.